src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +57,15 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        if (elements == null || element == null) {
+            throw new NoSuchElementException();
+        }
+        Stream<String> stream = elements.stream();
+        Optional<String> optional = stream.filter((n) -> n.equals(element)).findAny();
```

Comment:
thy to simplify and use method orElseThrow()

Reply:
OK.

Reply:
return elements.stream().filter((n) -> n.equals(element)).findAny().orElseThrow();